apps/shipments/filters.py

```diff
@@ -5,13 +5,25 @@
 from .models import Shipment, TransitState
 
 
+MULTI_STATE_CHOICES = tuple((m.name.lower(), m.value, ) for m in TransitState) + \
```

Comment:
We cannot really use the lookup expression parameter here because `MultipleChoiceFilter` expects a set of choices from which the passed in params are validated against. This check is independent from the django queryset that is being filtered. 
however I came out with an alternative solution, see `CaseInsensitiveMultipleChoiceField` and `CustomMultipleChoiceFilter` implementation in shipments.filters